rules/threat_intel/threat_intel_rapid7_threat_command.toml

```diff
@@ -0,0 +1,111 @@
+- Consider the history of the vulnerability that was matched. Has it happened before? Is it happening on multiple machines? These kinds of questions can help understand if the activity is related to legitimate behavior.
+- Consider the user and their role within the company: is this something related to their job or work function?
+"""
+references = [
+    "https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html",
+    "https://docs.elastic.co/integrations/ti_rapid7_threat_command"]
+risk_score = 99
+rule_id = "3a657da0-1df2-11ef-a327-f661ea17fbcc"
+setup = """
+
+## Setup
+
+This rule needs threat intelligence indicators to work.
+Threat intelligence indicators can be collected using an [Elastic Agent integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#agent-ti-integration),
+the [Threat Intel module](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#ti-mod-integration),
+or a [custom integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#custom-ti-integration).
+
+More information can be found [here](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html).
+
+## Max Signals
+
+This rule is configured to generate more **Max alerts per run** than the default 1000 alerts per run set for all rules. This is to ensure that it captures as many alerts as possible.
+
+**IMPORTANT:** The rule's **Max alerts per run** setting can be superseded by the `xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines the maximum alerts generated by _any_ rule in the Kibana alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule will still generate no more than 1000 alerts even if its own **Max alerts per run** is set higher.
+
+To make sure this rule can generate as many alerts as it's configured in its own **Max alerts per run** setting, increase the `xpack.alerting.rules.run.alerts.max` system setting accordingly.
+
+**NOTE:** Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless projects.
+"""
+severity = "critical"
+tags = [
+    "OS: Windows",
+    "Data Source: Elastic Endgame",
+    "Data Source: Windows",
+    "Data Source: Network",
+    "Data Source: Rapid7 Threat Command",
+    "Rule Type: Indicator Match",
+    "Resources: Investigation Guide",
+    "Use Case: Vulnerability",
+    "Use Case: Asset Visibility",
+    "Use Case: Continuous Monitoring",
+]
+threat_index = ["rapid7-tc-unique-cves"]
+threat_indicator_path = "rapid7.tc.vulnerability"
+threat_language = "kuery"
+threat_query = """
+@timestamp >= "now-30d/d" and vulnerability.id : * and event.module: ti_rapid7_threat_command
+"""
+timeline_id = "495ad7a7-316e-4544-8a0f-9c098daee76e"
+timeline_title = "Generic Threat Match Timeline"
+timestamp_override = "event.ingested"
+type = "threat_match"
+
+query = '''vulnerability.id : *'''
```

Comment:
It looks very specific, Does it match any of the provided indexes? Some of them are very high volume and could cause issues. Maybe we should include only vuln scan integrations?

Reply:
Great question - This is the original rule that is being ported over, it's been adjusted slightly based on my review of other threat intel rules.

https://github.com/elastic/integrations/blob/main/packages/ti_rapid7_threat_command/kibana/security_rule/eaecc8f0-6704-11ed-80b2-9bbc46f73b72.json

Reply:
Well, I think the users could also ingest events from the respective beats and the indices in question does seem to be appropriate match `index = ["auditbeat-*", "endgame-*", "filebeat-*", "logs-*", "packetbeat-*", "winlogbeat-*"]`. It also matches with existing rule already. 

Maybe for the `logs-*`, it is a bit broad and can be narrowed down, but I don't have a vuln scan integrations list ready. Also, we don't have any naming conventions on vulnerability integrations like we do `ti_*` for Threat Intel integrations, so this rule should keep changing to include newer integrations as they are added and maintaining it could be an issue.

Reply:
Thank you for the clarification, if a new index pattern is necessary to be agnostic with another integration we can always come back and tune the rule accordingly.